packages/amplify-cli-core/src/customPoliciesUtils.ts

```diff
@@ -103,14 +103,13 @@ function validateCustomPolicies(data: CustomIAMPolicies, categoryName: string, r
   const validatePolicy = ajv.compile(CustomIAMPoliciesSchema);
   const valid = validatePolicy(data);
   if (!valid) {
-    let errorMessage = `Invalid custom IAM policies in the ${resourceName} ${categoryName}.\n
-    Edit <project-dir>/amplify/backend/function/${resourceName}/custom-policies.json to fix
-    Learn more about custom IAM policies for ${categoryName}: https://docs.amplify.aws/cli/function/#access-existing-aws-resource-from-lambda-function\n`;
-    if (validatePolicy && validatePolicy.errors) {
-      errorMessage += validatePolicy.errors?.map((error: Ajv.ErrorObject) => error.message).join(EOL);
-    }
-
-    throw new CustomPoliciesFormatError(errorMessage);
+    printer.error(`${resourceName} ${categoryName} custom-policies.json failed validation:`);
+    formatter.list((validatePolicy?.errors || []).map(err => `${err.dataPath} ${err.message}`));
+    throw new CustomPoliciesFormatError(`
+      Invalid custom IAM policies for ${resourceName} ${categoryName}.
```

Comment:
Does this need an e2e test fix, since out messages have changed ?

Reply:
I searched for this string in our test suite and didn't get any hits, so hopefully we're good